src/components/Dropdown/Dropdown.tsx

```diff
@@ -117,7 +117,12 @@ export const Dropdown: FC<DropdownProps> = React.memo(
             );
         }, [refs.reference, refs.floating, update]);
 
-        useAccessibility(trigger, refs.reference, toggle(true), toggle(false));
+        useAccessibility(
```

Comment:
useAccessibility suggests this hook may be used for other keyboard  scenarios. This hook will need to support more than toggling the visibility of only this component.

Reply:
Suggest stubbing in a switch statement using this scenario as the first use case.

Reply:
this PR fixes a11y related issue in case of `portal=true`. 
we can take up more a11y support in the hook but it seems like a out of scope for this PR

Reply:
You should rename this hook to be more specific. Any refactor would put the scope of your hook at risk in the future.